pyvista/core/filters/poly_data.py

```diff
@@ -1634,12 +1642,27 @@ def decimate(
         tensors_weight : float, default: 0.1
             See ``scalars_weight`` parameter.
 
+        boundary_constraints: bool, default: False
+            Use the legacy weighting by boundary_edge_length instead of by
+            boundary_edge_length^2 for backwards compatibility
+
+            .. versionadded:: 0.45.0
+
+        boundary_weight: float, default: 1.0
+           A floating point factor to weigh the boundary quadric constraints
+           by: higher factors further constrain the boundary.
+
+            .. versionadded:: 0.45.0
```

Comment:
The docs may fail here, `..` should align with the text
```suggestion
           .. versionadded:: 0.45.0
```